appkit/src/main/java/org/ergoplatform/appkit/RestApiErgoClient.java

```diff
@@ -93,7 +103,26 @@ public static ErgoClient createWithoutExplorer(String nodeUrl, NetworkType netwo
      * @return a new instance of {@link ErgoClient} connected to a given node
      */
     public static ErgoClient create(String nodeUrl, NetworkType networkType, String apiKey, String explorerUrl) {
-        return new RestApiErgoClient(nodeUrl, networkType, apiKey, explorerUrl);
+        return new RestApiErgoClient(nodeUrl, networkType, apiKey, explorerUrl, null);
+    }
+
+    /**
+     * Creates a new {@link ErgoClient} instance connected to a given node of the given
+     *  network type.
+     *
+     * @param nodeUrl     http url to Ergo node REST API endpoint of the form
+     * `https://host:port/`
+     * @param networkType type of network (mainnet, testnet) the Ergo node is part of
+     * @param apiKey      api key to authenticate this client
+     * @param explorerUrl optional http url to Explorer REST API endpoint of the form
+     *                    `https://host:port/`. If null or empty, then explorer connection
+     *                    is not initialized so that the resulting {@link ErgoClient} can
+     *                    work in `node-only` mode.
+     * @param proxy       Requests are passed through this proxy (null it's ok).
```

Comment:
Same as above.

Additionally, it should be stated that proxy is only used for Explorer API, not node. Maybe the param should also be renamed to "explorerApiProxy" because of this.

By the way, we could start using `@Nullable` annotations here as well. Made a quick check and it seems javax.annotation.Nullable is on the classpath

Reply:
The mixer uses public nodes in the network, so a proxy is required to communicate with the nodes and the explorer.

Reply:
Sorry, did not see that proxy is used for node api as well.